docs/source/serialization.rst

```diff
@@ -168,7 +168,7 @@ register them with Dask.
 
     @dask_deserialize.register(Human)
     def deserialize(header: Dict, frames: List[bytes]) -> Human:
-        return Human(frames[0].decode())
+        return Human(frames[0].tobytes())
```

Comment:
This would need an `import codecs` above, but would work for a `memoryview` or a `bytes` object

```suggestion
        return Human(codecs.decode(frames[0]))
```

Reply:
Also just to demonstrate this works...

```python
In [1]: import codecs

In [2]: mv = memoryview(b"abc")

In [3]: codecs.decode(mv)
Out[3]: 'abc'
```